.goreleaser.yaml

```diff
@@ -29,3 +29,8 @@ changelog:
       - '^docs:'
       - '^test:'
       - '^Merge pull request'
+env:
+  - GOSUMDB=off 
```

Comment:
why?

Reply:
good question, I don't know but otherwise it fails with GOSUMDB not found, I think it's part of the change between go 1.19 and 1.22, and since our mod file are 1.19 based then it misses some of the configuration and you have to tune it off

Reply:
ummm this is weird, GOSUMDB should just point to the sum address like so `GOSUMDB='sum.golang.org'`

Reply:
can we maybe set

```
gomod:
  env:
    - GOSUMDB=sum.golang.org
 ```

Reply:
```suggestion
  - GOSUMDB=sum.golang.org
```

Reply:
how this made it work? lol I would expect that to be the default. Maybe goreleaser using a different empty go env?